Dockerfiles/agent/entrypoint.d/_default

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+/opt/datadog-agent/embedded/bin/fipsinstall.sh
```

Comment:
Things that need to be installed inside the agent docker image should be installed at image build time via `Dockerfile`.
The entrypoint will be executed each time a container is started.
This will fail on setups running with `ReadOnlyRootFilesystem: true` and on setups where the user isn’t `root`.
This will not be executed on setups where the default entrypoint is overridden.